demo/pymafx_estimate_smplx.py

```diff
@@ -366,5 +489,16 @@ def init_openpifpaf(parser):
         type=int,
         default=8,
         help='batch size for SMPL prediction')
+    if has_openpifpaf:
+        init_openpifpaf(parser)
     args = parser.parse_args()
+    if args.use_openpifpaf:
+        assert has_openpifpaf, 'Please install openpifpaf to run the demo.'
+        assert args.det_config is not None
+        assert args.det_checkpoint is not None
+    if args.use_mmpose:
+        assert has_mmdet, 'Please install mmdet to run the demo.'
+        assert has_mmpose, 'Please install mmpose to run the demo.'
+        assert args.det_config is not None
+        assert args.pose_config is not None
```

Comment:
Shall we `assert args.use_mmpose or args.use_openpifpaf`?

Or shall we simply remove `use_mmpose`? If users do not pass `use_openpifpaf`, we choose mmpose automatically.

Reply:
I removed `use_mmpose`, and set mmpose as the default.